payments-core/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/TransformToPaymentMethodCreateParams.kt

```diff
@@ -112,3 +138,6 @@ class TransformFormToPaymentMethod {
         }
     }
 }
+
+@Suppress("UNCHECKED_CAST")
+private fun <K, V> Map<K, V?>.filterOutNullValues() = filterValues { it != null } as Map<K, V>
```

Comment:
I think `filterNotNullValues` would be more idiomatic

Reply:
sorry, I am happy to make the update, are you suggesting

private fun <K, V> Map<K, V?>.filterOutNullValues() = filterNotValues { it == null } as Map<K, V>

or

private fun <K, V> Map<K, V?>.filterNotNullValues() = filterNotValues { it == null } as Map<K, V>

@mshafrir-stripe

Reply:
In Kotlin, `filterX` means "only give me X", so I thought that `filterNotNullValues` would be what you want to call a method that only gives you not-null values